src/lib/teamInfo.js

```diff
@@ -94,26 +103,39 @@ const teamInfo = {
   ],
   CoIs: [
     {
-      name: 'Manuel A. Rivas, Assistant Professor',
+      name: 'Manuel A. Rivas, Ph.D.',
       title: 'Co-Investigator',
       image: MannyImg,
     },
     {
-      name: 'Rob Tibshirani, Professor, Ph.D.',
+      name: 'Rob Tibshirani, Ph.D.',
       title: 'Co-Investigator',
       image: RobImg,
     },
     {
-      name: 'Trevor Hastie, Professor, Ph.D.',
+      name: 'Trevor Hastie, Ph.D.',
       title: 'Co-Investigator',
       image: TrevorImg,
     },
     {
-      name: 'Kelly E. Ormond, MS, CGC',
+      name: 'Kelly E. Ormond, M.S., CGC',
       title: 'Co-Investigator',
       image: KellyImg,
     },
+    {
+      name: 'Jonathan N. Myers, Ph.D.',
+      title: 'Advisor',
+      image: JonImg,
+    },  
   ],
+  Alumni: [
+    {
+      name: 'Samir Akre',
+      title: 'Alumni',
+      //title: 'Alumni Software & Pipeline Developer',      
```

Comment:
@ysyskim We generally don't retain unused code in comments. `git` gives us the ability to roll back changes if needed.

Reply:
This comment has been removed.

Reply:
Thanks!